applications/vessel-history/src/types/index.ts

```diff
@@ -21,40 +21,35 @@ export type MapCoordinates = {
   longitude: number
   zoom: number
 }
-export interface Flag {
-  start: string
-  end: string
-  value: string
-}
 
-export interface Mmsi {
-  start: string
-  end: string
-  value: string
+export interface OtherCallsign {
+  counter: number
+  name: string
 }
-export interface Callsign {
-  start: string
-  end: string
-  value: string
+
+export interface OtherImo {
+  counter: number
+  name: string
 }
-export interface Extra {
-  id: string
-  label: string
-  value: string
+
+export interface OtherShipname {
+  counter: number
+  name: string
 }
+
 export interface Vessel {
   id: string
-  vesselId: string
-  type: string
-  ssvid: string
-  name: string
-  imo: string
-  flags: Flag[]
-  authorizations: any[]
-  mmsi: Mmsi[]
-  callsign: Callsign[]
-  extra: Extra[]
+  callsign: string
```

Comment:
~isn't this a breaking change with the old API~ ? I just realised not depends on the API anymore but on the dataset so I guess we'll need to support generics here